CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## Improvements
 
 ## Bug Fixes
+- PR #1319 Fix graph nodes function and renumbering from series
```

Comment:
Doesn't this get auto populated now from the PR title.  Perhaps the PR title should be:  `Fix graph nodes function and renumbering from series` and this line shouldn't be here?